swagger.yaml

```diff
@@ -4470,6 +4475,53 @@ paths:
           schema:
             $ref: '#/definitions/error'
 
+  /v2/infra-envs/{infra_env_id}/downloads/ipxe-script-url:
```

Comment:
We already have:
/v2/clusters/{cluster_id}/downloads/files-presigned
So maybe create
/v2/infra-envs/{infra_env_id}/downloads/files-presigned
?

Reply:
:shrug: Sure.

Do we care about making it work for the discovery ignition as well as the ipxe script?